### ISSUE TITLE
Need proper error message when config.yml has syntax errors 

### ISSUE THREAD

**Issue by amolkahat** [geeksocket/review-board#6]
```
akahat@akahat-pnq-redhat-com review-board $ python3 app.py 
Welcome to Code Review System

The Application will read username and code sites from configuraiton file
Traceback (most recent call last):
  File "app.py", line 37, in <module>
    execute_board()
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "app.py", line 18, in execute_board
    conf = yaml.load(f)
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/yaml/__init__.py", line 72, in load
    return loader.get_single_data()
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/yaml/constructor.py", line 35, in get_single_data
    node = self.get_single_node()
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/yaml/composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/yaml/composer.py", line 82, in compose_node
    node = self.compose_sequence_node(anchor)
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/yaml/composer.py", line 110, in compose_sequence_node
    while not self.check_event(SequenceEndEvent):
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/home/akahat/Templates/src/review-board/review_board/lib/python3.5/site-packages/yaml/parser.py", line 393, in parse_block_sequence_entry
    "expected <block end>, but found %r" % token.id, token.start_mark)
yaml.parser.ParserError: while parsing a block collection
  in "config.yml", line 2, column 5
expected <block end>, but found '?'
  in "config.yml", line 3, column 5

```

**Comment by anshu686** [comment]
Hi, Can I work on this issue?

**Comment by bhavin192** [comment]
Fixed by #11

**Comment by bhavin192** [comment]
Fixed by #11
